roles/install-app/tasks/main.yml

```diff
@@ -31,6 +32,14 @@
     - install
     - app
 
+# mock does work on CentOS. Since it is used only for testing, we can remove it
```

Comment:
Hmmm... I don't really like this approach.
Since installing `mock` in production doesn't make sense also on Debian, it would be better to split the `requirements.txt` file into separate prod and dev ones.

A `requirements.txt` file supports also the inclusion of other files, so we could have a generic `requirements.txt` with all the production values, and a `requirements-dev.txt` that is made like this:
```
-r requirements.txt
mock
```

Installation notes should be updated as well.